File: Python/Problem-14.py
# Write A Program To Calculate The Simple Intrest

# Using Function
# def simple_Intrest(p,r,t):
#   print("The Princepal Amount",p)
#   print("The Interest rate",r)
#   print("The Time",t)

#   si = (p * r * t)/100
#   print("The si is :" , si)
#   return si
# simple_Intrest(30000,5,10)

# Using Normal Way
p = float(input("Enter Any Number: "))
r = float(input("Enter Any Number: "))
t = int(input("Enter Any Number: "))

si = (p * r * t)/100
print("The Simple Intrest is : ", si)

File: JavaScript/Problem-14.js
// Write a Program To Calculate The Simple Intrest
const prompt = require("prompt-sync")();
var p = parseFloat(prompt("Enter Any Number: "));
var r = parseFloat(prompt("Enter Any Number: "));
var t = parseInt(prompt("Enter Any Number: "));

const si = (p * r * t) / 100;
console.log("The Simple Intrest is : ", si);
